src/java.desktop/share/classes/javax/sound/sampled/AudioSystem.java

```diff
@@ -238,17 +237,11 @@ public static Mixer getMixer(final Mixer.Info info) {
 
             currentInfoArray = mixer.getSourceLineInfo(info);
             for (int j = 0; j < currentInfoArray.length; j++) {
-                vector.addElement(currentInfoArray[j]);
+                list.add(currentInfoArray[j]);
             }
         }
 
-        Line.Info[] returnedArray = new Line.Info[vector.size()];
-
-        for (int i = 0; i < returnedArray.length; i++) {
-            returnedArray[i] = vector.get(i);
-        }
-
-        return returnedArray;
+        return list.toArray(new Line.Info[0]);
```

Comment:
I thought we already covered such changed under JDK-8269130, is there are any other missed cases? If yes then probably it will be good to extract such changes.

Reply:
In JDK-8269130 I fixed only usages of Collections.toArray. and didn't touch manual copying.
Reverted usage of toArray where it wasn't used before.